<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournaldConfigurationWidget</class>
 <widget class="QWidget" name="JournaldConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Journald options</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="currentUserEntries">
        <property name="text">
         <string>Display log entries of processes of the current user</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="lastBootOnly">
        <property name="text">
         <string>Display log entries from current boot only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="systemEntries">
        <property name="text">
         <string>Display log entries of system services and the kernel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
